app/assets/stylesheets/content/_forms.lsg

```diff
@@ -178,6 +178,16 @@
         The more, the better
       </div>
     </div>
+    <div class="form--field">
+      <label class="form--label">Telephone:</label>
+      <div class="form--field-container">
+        <div class="form--field-affix icon-context icon-phone">
+        </div>
+        <div class="form--text-field-container">
+          <input type="number" class="form--text-field">
```

Comment:
:+1: 

Lets do that ^^

(after all, if a browser does not support it, it will fall back to `text`)